Assets/Engine/Inventory/Editor/ContainerDescriptorEditor.cs

```diff
@@ -304,7 +307,15 @@ private void HandleHideItems(bool hideItems)
     private void AddInteractive()
     {
         SerializedProperty sp = serializedObject.FindProperty("containerInteractive");
-        sp.objectReferenceValue = containerDescriptor.gameObject.AddComponent<ContainerInteractive>();
+        GameObject networkedParent = GetParentNetworkIdentity(containerDescriptor.gameObject);
+        if (networkedParent != null)
+        {
+            sp.objectReferenceValue = networkedParent.AddComponent<ContainerInteractive>();
+        }
+        else
+        {
+            sp.objectReferenceValue = containerDescriptor.gameObject.AddComponent<ContainerInteractive>();
+        }
```

Comment:
Single line if/else's should be inlined imo:
```
sp.objectReferenceValue = networkedParent != null ?
networkedParent.AddComponent<ContainerInteractive>() : 
containerDescriptor.gameObject.AddComponent<ContainerInteractive>();
```

Reply:
done